dptscan
=================
* Introduction
dptscan implemented the iDPT method(1,2), which integrates signal Deconvolution, whole genome Pattern recognition and scan, and differential Testing into a consolidated pipeline, to address the group-wise comparison of genome-wide, enrichment-based profiles generated by massively parallel sequencing.
dptscan website: http://idpt.github.com/dptscan/

* Install
Check the dptscan wetsite for system requirement
1) Follow the data preparation steps listed on the dptscan website.
2) Create a folder as project workspace. Copy or link the preprocessed data profiles from 1) to the data subfolder InputData (default).
3) Download the zip files from the dptscan website and unzip all the files into the project workspace folder.
4) Configure the samplesheet.text and experiment-config.r files.
5) Make dptscan excitable with 'chmod +x dptscan".
6) Initialize/update the workspace with "dptscan -s".

* Usage of dptscan
Usage: dptscan options
Example: ./dptscan --batch=1,1,5,5,5 --core=1,10,4,4,4

Options:
  --verbose        show more onscreen information.
  -s               setup dptscan workspace.
  --task=TASK      select one or more modules from "preprocess", "mixPoi", "pattRecog", "diffTest", "report". DEFAULT:"preprocess, mixPoi,  pattRecog,  diffTest, report", and this performs the complete dpscan process with all the modules applied in a proper order.
  --chr=CHR        select an individual or a group of chromosomes to run. DEFAULT:1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,X,Y,M.
  --core=CORE      number of computer processers to use in parallel for each batch.
  --batch=BATCH    number of batch tasks in parallel.
  --samplesheet=SHEET    
                   sample sheet file. DEFAULT: "samplesheet.txt".
  --config=CONFIG  configuration file for running parameters.
  --output=PATH    path to output result files. DEFAULT: "./", current location.
  --analysis=methyl
                   type of analysis. DEFAULT: methyl, DNA methylation analysis.

